### DISCUSSION TITLE
Search SDK in React Native

### DISCUSSION THREAD

**Discussion by pavlee** [rnmapbox/maps#1626] [Q&A]
I have a requirement to develop a map view with a search box with autocomplete to search places by address. I found [this page](https://docs.mapbox.com/ios/search/guides) on Mapbox website, but it is available only in native code.

I've seen the wrapper for this SDK doesn't exist in this repository, and I'm wondering if it is possible to create a library for react native? I never used native modules in React, nor do I have any experience with Swift, and just want to know if it's possible to import this API and native components into my project?

**Comment by ferdicus** [comment]
Yes, it is possible.  
Do we see the need or do we have the people power to do that? No.  
Feel free to investigate and open a PR.  

Another way would be to use a JS solution
